src/Sarif.FunctionalTests/SarifConverterTestData/AndroidStudio/AndroidStudio_issueLog1_raw.xml.sarif

```diff
@@ -17,6 +17,36 @@
           }
         ]
       },
+      "logicalLocations": {
+        "app\\microsoft.androidstudioformatexample\\microsoft.androidstudioformatexample.ApplicationTest ApplicationTest()": [
+          {
+            "name": "app",
+            "kind": "android-module"
```

Comment:
OK, I see that the schema text for `logicalLocation.kind` says:

> `Should be one of 'assembly', 'attribute', 'binary', 'class', 'enum', 'field', 'lambda', 'local', 'method', 'module', 'namespace', 'package', 'parameter', 'property', 'resource', or, 'struct', if any of those accurately describe the construct.`

Neither the spec text nor the `LogicalLocationKind` enum has been brought up to date with that text. I'll take care of it.